python/lsst/daf/butler/formatters/parquet.py

```diff
@@ -915,7 +919,7 @@ def _standardize_multi_index_columns(
     pd_index = arrow_schema_to_pandas_index(schema)
     index_level_names = tuple(pd_index.names)
 
-    names = []
+    names: List[Union[str, Sequence[Any]]] = []
```

Comment:
You can now say:

```python
names: list[str | Sequence[Any]] = []
```
(ie use `|` instead of `Union` and `list` instead of `List`).